src/Pages/Home/Home.js

```diff
@@ -2,10 +2,10 @@ import React, { Component } from 'react'
 import './Home.css'
```

Comment:
Rename `Home.css` to `home.css`
```suggestion
import './home.css'
```